docs/release_notes/ibexa_dxp_v4.6.md

```diff
@@ -4,49 +4,446 @@ description: Ibexa DXP v4.6 brings improvements to Commerce, PIM and Personaliza
 
 <!-- vale VariablesVersion = NO -->
 
-# Ibexa DXP v4.6
+[[= release_notes_filters('Ibexa DXP v4.6 LTS', ['Headless', 'Experience', 'Commerce', 'LTS Update', 'New feature']) =]]
 
-**Version number**: v4.6
+<div class="release-notes" markdown="1">
 
-**Release date**: February 13, 2024
+[[% set version = 'v4.6.18' %]]
+[[= release_note_entry_begin("Ibexa DXP " + version, '2025-03-06', ['Headless', 'Experience', 'Commerce']) =]]
```

Comment:
I was wondering:
1) whether we need edition pills next to each release number. Was there ever a version released for less than three editions? I'd say this is redundant information.
2) why these pills are different from the pills visible when within the article below, we should be consistent visually
3) do we need "New feature" pill if right below the pills we have the "Features" header. I'd say not, because if a version has new features, it is very well visible because their list is right below the version number.

IMVHO there's too much clutter, too many colors.

![image](https://github.com/user-attachments/assets/4bbaf7cd-419c-4dc7-b7bf-a07f2af2f45c)

Reply:
1) We need them for the filters to work, and if we release an LTS update available not available for Headless then it would be useful. I'd keep it and we can revisit it in the future if needed.
2) I've discussed this with @GrabowskiM - we will align the pills visually once this is merged to keep the scope of this PR small
3) I'd keep it as it works well with filtering - you can have a quick a glance of what was added in the recent releases